packages/js-dpp/lib/test/mocks/createStateRepositoryMock.js

```diff
@@ -62,4 +62,8 @@ module.exports = function createStateRepositoryMock(sinonSandbox) {
     fetchLatestPlatformBlockTime: sinonSandbox.stub(),
     calculateStorageFeeDistributionAmountAndLeftovers: sinonSandbox.stub(),
```

Comment:
We can remove this one. It's not used anymore.

Reply:
Fixed